### ISSUE TITLE
Grid Filters and Commands section 508 compliance 

### ISSUE THREAD

**Issue by Alex-Bubblemaster** [telerik/kendo-ui-core#3388]
### Enhancement

As suggested in ticket ID: 1117900

- Within an `<a>` tag, there must be text or an image with `alt` property.
- Each `<a>` tag must have a unique `title` if text or image is the same for more than one `<a>` tag.
- Since command buttons are rendered as `<a>` tags, they should also have a unique `title`. 

<blockquote>
The title "Filter" will NOT satisfy the need if there is more than one filter.  It must read "Filter country by size" or "Filter state by population". Each title for an `<a>` tag within a page must be unique.
</blockquote>

### Reproduction of the problem

![image](https://user-images.githubusercontent.com/10980842/27957898-1b0fb6fe-6329-11e7-91ff-6afc74add5b9.png)

### Current behaviour

All filter titles are the same. The command buttons have no `titles`.

### Expected/desired behavior

Filter titles should be distinct and descriptive. Command buttons should obey the same rules.

### Environment

* **Kendo UI version:** 2017.2.621
